BE/OwnershipAndControl/Executives.rdf

```diff
@@ -475,17 +483,40 @@
 		<skos:definition>indicates a party to which this legal person has delegated some authority or control</skos:definition>
 	</owl:ObjectProperty>
 	
+	<owl:ObjectProperty rdf:about="&fibo-be-oac-exec;designatesSignatory">
+		<rdfs:subPropertyOf rdf:resource="&fibo-be-oac-exec;authorizes"/>
+		<rdfs:label>designates signatory</rdfs:label>
+		<rdfs:range rdf:resource="&fibo-be-oac-exec;Signatory"/>
+		<skos:definition>authorizes a responsible party to sign agreements, access accounts and/or perform other similar tasks on their behalf</skos:definition>
+	</owl:ObjectProperty>
+	
 	<owl:ObjectProperty rdf:about="&fibo-be-oac-exec;elects">
 		<rdfs:label>elects</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-fnd-pty-pty;PartyInRole"/>
-		<rdfs:range rdf:resource="&fibo-fnd-pty-pty;PartyInRole"/>
+		<rdfs:domain rdf:resource="&fibo-be-oac-exec;AuthorizingParty"/>
+		<rdfs:range rdf:resource="&fibo-be-oac-exec;AuthorizedParty"/>
 		<skos:definition>chooses someone to hold office or some other position by voting</skos:definition>
 		<skos:editorialNote>In the case of an election for a member of the board of directors, the bylaws state the manner in which that process is effected. The candidate members may be recommended by the board or other proxy and are then elected by the shareholder. A similar process may be conducted to elect outside auditors.</skos:editorialNote>
```

Comment:
shareholder should be plural. 
For a public company with millions of shareholders would each be linked to a BoardMember using this property, assuming they voted? I don't FIBO is intended for use to tally votes!
As per previous comment need to distinguish appoints from elects.
Overall I don't think this property is needed at all beyond a general need to track where someone has control and appoints a director.

Reply:
fixed and changed back to party-in-role, since I think one elects a slate of directors at once in some cases, and individual directors in other cases

Reply:
You didn't answer my question about how this would ever be used and whether each (voting?) Shareholder would be linked? What if one shareholder voted No or Abstained or gave their Proxy to the company secretary? Not that I think FIBO should be used for vote tallying: what is the point of this property?